src/service-card.js

```diff
@@ -0,0 +1,71 @@
+import { LitElement, html, css } from 'lit-element/lit-element.js';
+
+class ServiceCard extends LitElement {
+  static get tag() {
+    return 'service-card';
+  }
+
+  static get properties() {
+    return {
+      image: { type: String },
+      title: { type: String },
+      info: { type: String },
+      alt: { type: String },
+      uniformPadding: { type: String, attribute: 'uniform-padding' },
+    };
+  }
+
+  constructor() {
+    super();
+    this.title = 'A shorter title';
+    this.info = 'Here is the information about the thing';
+    this.uniformPadding = '2';
```

Comment:
you dont actually use this value so should look at doing so. Also if it's a numerical value then should be typed number and without `' '` around it